config/config.go

```diff
@@ -65,6 +65,14 @@ const (
 	DefTableColumnCountLimit = 1017
 	// DefMaxOfTableColumnCountLimit is maximum limitation of the number of columns in a table
 	DefMaxOfTableColumnCountLimit = 4096
+	// DefStatsLoadConcurrencyLimit is limit of the concurrency of stats-load
+	DefStatsLoadConcurrencyLimit = 1
```

Comment:
Does this variable mean the minimum value? Maybe `DefMinOfStatsLoadConcurrencyLimit`or `DefMinStatsLoadConcurrency` is ok?

Reply:
Just to be consistent with other limits...